locations/spiders/lidl_de.py

```diff
@@ -14,18 +17,36 @@
     "So": "Su",
 }
 
+HEADERS = {"X-Requested-With": "XMLHttpRequest"}
+STORELOCATOR = "https://spatial.virtualearth.net/REST/v1/data/ab055fcbaac04ec4bc563e65ffa07097/Filialdaten-SEC/Filialdaten-SEC?$select=*,__Distance&$filter=Adresstyp%20eq%201&key=AiUPPcYK4rdGkNaJ9FLpcgcgcEfzdPNcqEafbU8B6m8hzjcQk_urZxCWtLvFsHSq&$format=json&jsonp=Microsoft_Maps_Network_QueryAPI_3&spatialFilter=nearby({:},{:},40.1931215)"
```

Comment:
If you drop the `jsonp` param from the URL you probably get real json back

Reply:
It returns an error when I remove it

Reply:
But it works if you drop spatialFilter, weird. You can check out lidl_gb if you want to page through the api instead of lat/lon search.

I might write some shared code to parse virtualearth anyway tomorrow or next week.

Reply:
Ok I will have a look at it later.

Reply:
See #4419